src/styles.scss

```diff
@@ -1,21 +1,21 @@
-#EZDrawer {
-    #EZDrawer__checkbox {
+.EZDrawer {
```

Comment:
Even if you have multiple drawers now it will work the same as before.

imagine there are two drawers on the screen with the same classNames.

Reply:
and ? the problem was with the htmlFor, that will target an Id . not a className https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for